README.md

```diff
@@ -1,37 +1,79 @@
+The `CageService` implements a Windows service running in the background. It receives messages from the `CageChooser` and sends messages to the `CageManager`.
 
 ### CageManager
 
-The `CageManager` creates a new desktop and processes in it, according to the messages received by the `CageService`.
+The `CageManager` creates a new desktop and starts the program and optionally the additional program according to the config received from the `CageChooser` over the `CageService`. In addition, the icon and some additional informations are displayed. Using the displayed "Activate"-button(s) the program(s) can be restarted or brought back into the foreground.
+
+<details><summary markdown="span"><code>Fold out to see sample</code></summary>
+
+<img width="1680" alt="Screenshot CageManager" src="https://user-images.githubusercontent.com/1786772/43678602-914198ec-9816-11e8-8a75-0ac3032368e4.png">
+
+</details>
+
+### CageChooser
+
+The `CageChooser` is a user interface which displays in a list all available configs on the system by iterating over the registry entries under the following path: `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\SharkCage\Configs`. By selecting a config and pressing the "Start"-button (or the enter key), the `CageManager` will be started. As soon as the `CageManager` is started, the `CageChooser` will send the path to the selected config to the `CageManager`.
+
+<details><summary markdown="span"><code>Fold out to see sample</code></summary>
 
-### StarterCmd
+<img width="500" alt="Screenshot CageChooser" src="https://user-images.githubusercontent.com/1786772/43733677-c50f6b72-99b5-11e8-9831-69556fd33246.png">
 
-A user interface for starting the `CageManager` and an application in it. Directly communicating with the `CageService`.
+</details>
 
-### ImageSelectDialog
+### CageConfigurator
 
-A user interface to select an icon to be displayed on the user's desktop. `config.txt` will be created in `C:\sharkcage` with a line containing `picture: ` followed by the index of the icon. The `CageService` reads this configuration file and searches for that line and opens the dialog if the file contains no such line.
+The `CageConfigurator` is a user interface to create a config file including an icon to be displayed on the secure desktop, the program which should be started and an optionally additional program. The additional program can be choosen out of a list of "trustworthy" applications.
+ As soon as the config file has been saved, a link to the config which is stored under `C:\Users\Public\Documents\SharkCage\` will be saved in the Registry under the following path: `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\SharkCage\Configs`.
+ The config contains json data and is protected using ACL's to be read and edited only by the `CageConfigurator` in a secure environment, the `CageService` as well as the administrator group.
```

Comment:
'The config contains json data and is protected using ACL's to be read and edited only by the `CageConfigurator` in a secure environment, the `CageService` as well as the administrator group.'
->
'The confic contains json data and its access rights restrict anyone except the administrator group from accessing the file in any way.'